### ISSUE TITLE
Mobile Friendly Layout / Small Screen

### ISSUE THREAD

**Issue by magdesign** [wwmm/easyeffects#2955]
Linux mobile user here, i love easyeffects to adjust the sound to headphones.

However, its quite hard to equalize on mobile screen/small screen with scaling. Would be nice if we could hide/minimize the left side pane.

Here some screenshots:

![2024-03-04-20:12:43](https://github.com/wwmm/easyeffects/assets/5273166/c33d4bd5-ea56-4060-97bd-b167674f817f)

and portrait:

![2024-03-04-20:02:29](https://github.com/wwmm/easyeffects/assets/5273166/0a017ba0-f20c-4398-862c-2d18f47c366d)

**Comment by violetmage** [comment]
Just to clarify, this is a feature request, yes?

I say this because you haven't pointed anything out that's technically _wrong_ with the behavior / display, just that it doesn't all fit on such a small screen.

There are probably a number of good solutions to this problem, but unless I'm missing something, all of them would require a non-trivial amount of code rework...

**Comment by wwmm** [comment]
>There are probably a number of good solutions to this problem, but unless I'm missing something, all of them would require a non-trivial amount of code rework...

I am afraid so. The library libadwaita we use offers some level of automation but they are all for applications that do not have as much functionality to expose as EasyEffects. We will need deep changes to our user interface just to make make things more bearable. Making it good will be even harder...

Out of curiosity which mobile system is this @magdesign ? I think it is the first time I see someone using EE on mobile.

**Comment by Digitalone1** [comment]
I thought libadwaita had the purpose of being good on small screens. But I'm afraid it's not that good because when I saw their widgets, there wasn't nothing that automatically switch from horizontal to vertical layout.

We need to nest group of controls inside multiple horizontal containers that auto-align vertically when the screen width gets small.

I don't know at the moment, but month ago I didn't find anything like that in libadwaita, only something that autohide when the screen gets small, which is not good is our use case.

**Comment by magdesign** [comment]
@violetmage 
Yes, its a feature request. Sorry for not clarifying... was not sure myself 😂

@wwmm 
postmarketos, sxmo, sway, oneplus6 phone.
scaling set to 3. 
equalizer is essential to enhance standard music quality.... 

linuxmobile is the new freedom 😛

**Comment by wwmm** [comment]
> @violetmage Yes, its a feature request. Sorry for not clarifying... was not sure myself 😂
> 
> @wwmm postmarketos, sxmo, sway, oneplus6 phone. scaling set to 3. equalizer is essential to enhance standard music quality....
> 
> linuxmobile is the new freedom 😛

I see. I did not know this was even possible :smile: . I am not going to lie. It is going to be really hard to do this. If even possible... Specially because none of the few people regularly contributing with code has the system to do tests. We would be shooting in the dark...

**Comment by wwmm** [comment]
Our QT branch is able to deal with small screen sizes a lot better

<img width="726" height="855" alt="Image" src="https://github.com/user-attachments/assets/ed0ee519-0b6e-4628-a58c-f8b49072277b" />
. It is not perfect but I think we can close this issue.

**Comment by magdesign** [comment]
thank you!
